scripts/kurtosis/run.sh

```diff
@@ -0,0 +1,224 @@
+    echo -e "${GREEN}[INFO]${NC} $1"
+}
+
+print_error() {
+    echo -e "${RED}[ERROR]${NC} $1"
+}
+
+print_warn() {
+    echo -e "${YELLOW}[WARN]${NC} $1"
+}
+
+# Function to check if command exists
+command_exists() {
+    command -v "$1" >/dev/null 2>&1
+}
+
+# Function to check dependencies
+check_dependencies() {
+    print_info "Checking dependencies..."
+    
+    if ! command_exists kurtosis; then
+        print_error "Kurtosis is not installed. Please install it from https://docs.kurtosis.com/install"
+        exit 1
+    fi
+    
+    if ! command_exists docker; then
+        print_error "Docker is not installed. Please install Docker."
+        exit 1
+    fi
+}
+
+# Function to start the testnet
+start_testnet() {
+    local config_path="$1"
+    
+    print_info "Starting testnet with enclave name: $ENCLAVE_NAME"
+    
+    # Check if enclave already exists
+    if kurtosis enclave inspect "$ENCLAVE_NAME" >/dev/null 2>&1; then
+        print_warn "Enclave $ENCLAVE_NAME already exists. Cleaning it up..."
+        kurtosis enclave rm -f "$ENCLAVE_NAME"
+    fi
+    
+    # Run Kurtosis with the configuration
+    print_info "Starting Kurtosis with configuration: $config_path"
+    kurtosis run --enclave "$ENCLAVE_NAME" "$KURTOSIS_PACKAGE" --args-file "$config_path"
+    
+    print_info "Testnet started successfully!"
+    print_info "View services: kurtosis enclave inspect $ENCLAVE_NAME"
+}
+
+# Function to stop the testnet
+stop_testnet() {
+    print_info "Stopping testnet: $ENCLAVE_NAME"
+    kurtosis enclave stop "$ENCLAVE_NAME"
+    
+    print_info "Cleaning up testnet resources..."
+    kurtosis enclave rm -f "$ENCLAVE_NAME"
+    
+    print_info "Testnet stopped and cleaned up successfully"
+}
+
+# Function to clean up the testnet
+clean_testnet() {
+    print_info "Cleaning up testnet: $ENCLAVE_NAME"
+    kurtosis enclave rm -f "$ENCLAVE_NAME"
+}
+
+# Function to show logs
+show_logs() {
+    local service="$1"
+    if [ -z "$service" ]; then
+        print_info "Available services:"
+        kurtosis service ls "$ENCLAVE_NAME"
+    else
+        print_info "Showing logs for service: $service"
+        kurtosis service logs "$ENCLAVE_NAME" "$service" --follow
+    fi
+}
+
+# Function to port-forward a service
+port_forward() {
+    local service="$1"
+    local port="$2"
+    
+    if [ -z "$service" ] || [ -z "$port" ]; then
+        print_error "Usage: $0 port-forward <service-name> <port>"
+        exit 1
+    fi
+    
+    print_info "Port-forwarding $service:$port"
+    kurtosis port-forward "$ENCLAVE_NAME" "$service" "$port"
+}
+
+# Function to build local Docker image
+build_local() {
+    print_info "Building local Lodestar Docker image..."
+    
+    # Get the root directory (two levels up from scripts/kurtosis)
+    # TODO: There might be a better way to get the root
+    LODESTAR_ROOT="$SCRIPT_DIR/../.."
+    
+    # Build the Docker image
+    if docker build -t lodestar:local "$LODESTAR_ROOT"; then
```

Comment:
for faster local builds
```suggestion
    if docker build -f Dockerfile.dev -t lodestar:local "$LODESTAR_ROOT"; then
```